### PR TITLE
WIP. [MSGINA] gui: Improve DLG_DATA handling. CORE-16775

### PR DESCRIPTION
**Pull request by SergeGautherie**
## Purpose

Trying to make it a bit clearer/safer.

JIRA issue: [CORE-16775](https://jira.reactos.org/browse/CORE-16775)

## TODO

- [ ] @katahiromz, you added `LogoWidth` and `LogoHeight` on 623dd26ccea5e79001aee769c230fb4f8e304fc8.
Can they be removed? Or what are they intended for?
- [ ] @julcar, could you test this PR? Thanks.

### CODE REVIEW

**Comment by katahiromz** [comment]
Logo height is used for dialog layout adjustments of Chinese, Japanese and Korean (CJK; far-east Asian) versions that differ in font size thus in dialog extent.
The CJK version of the dialog box is different in its size.

File: dll/win32/msgina/gui.c [hunk 1]
```diff
@@ -210,7 +210,7 @@ StatusDialogProc(
             DlgData_LoadBitmaps(pDlgData);
             if (pDlgData->hBarBitmap)
             {
-                pDlgData->TimerID = SetTimer(hwndDlg, -1, 20, NULL);
+                pDlgData->TimerID = SetTimer(hwndDlg, -1, 200, NULL);
```
Comment (Fraizeraust) [inline dll/win32/msgina/gui.c]: Please don't change stuff that actually do not bring *improvements* at all. The elapse time parameter is set to this value to nearly match the speed of the bar movement rotation in Windows Server 2003's MSGINA.

Comment (SergeGautherie) [reply dll/win32/msgina/gui.c:217]: Noted. Meanwhile, I am interested in slowing down as a test.

**Comment by SergeGautherie** [comment]
> Logo height is used

Noted. Could you briefly explain to me how that works, as in where/how is this variable(s) used? Thanks.

**Comment by katahiromz** [comment]
The dialog item is measured in dialog units but the logo image extent is measured in pixels.

**Comment by SergeGautherie** [comment]
> positioned

Sorry, let me ask differently:
which code (result) is broken by removing these variables?

As far as I can tell, they are unused.
https://git.reactos.org/?p=reactos.git&a=search&h=HEAD&st=grep&s=LogoWidth|LogoHeight&sr=1

**Comment by JoachimHenze** [comment]
Tons of commits in here. Which one does fix the issue exactly?

**Comment by SergeGautherie** [comment]
> Which one does fix the issue exactly?

I don't know! Why don't you tell us?

**Comment by JoachimHenze** [comment]
Closing this PR, as CORE-16775 was reported to be working now.
Some of the changes in PR2490 are not desireable.
The changes in here got superseded by PR2505 + PR2518.
#2505 was committed in [0.4.14-dev-1353-ga5a426f|https://git.reactos.org/?p=reactos.git;a=commit;h=a5a426f867c6ef160e083e2477bc815bd0acaebf]
and #2518 was committed in [0.4.14-dev-1354-g28e277b|https://git.reactos.org/?p=reactos.git;a=commit;h=28e277b65f236393cc84e3a8795a9e9e0846b811]

**Pull request closed without merging**